manifest_staging/charts/aad-pod-identity/templates/mic-exception.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.installMICException }}
+{{- if and (.Values.installMICException) (eq .Values.operationMode "standard") }}
```

Comment:
Let's wrap only the `mic` exception inside standard mode check. The aks-exception user should be able to disable/enable if aks/non-aks cluster.

```
{{- if .Values.installMICException }}
{{- if eq .Values.operationMode "standard" }}
<install MIC exception>
{{- end }}
<install aks exception>
{{ - end }}
```